src/renderer/icon-themes/base64-theme.ts

```diff
@@ -13,18 +13,19 @@ import { Base64Picker } from './icon-pickers/base64-picker';
 
 /**
  * This class implements an icon theme that allows the user to enter base64 encoded images
- * directly.
+ * or URLs directly.
  */
 export class Base64Theme implements IIconTheme {
   /** Returns a human-readable name of the icon theme. */
   get name() {
-    return 'Base64';
+    return 'Base64 or URL';
```

Comment:
Here we could maybe use this to avoid needing localization:

```suggestion
    return 'Base64 / URL';
```